MAUI/DataGrid/grid-events.md

```diff
@@ -0,0 +1,201 @@
+    var column = e.Column;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellRightTapped event
+This event will be triggered when a right-click mouse gesture is recognized on a cell. This event has the [DataGridCellRightTappedEventArgs](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.DataGrid.DataGridCellRightTappedEventArgs.html) as arguments.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellRightTapped="SfDataGrid_CellRightTapped"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void SfDataGrid_CellRightTapped(object sender, Syncfusion.Maui.DataGrid.DataGridCellRightTappedEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+    var pointerDeviceType = e.PointerDeviceType;
+}
+{% endhighlight %}
+{% endtabs %}
+
+N> The `CellRightTapped` event is only applicable for Windows and macOS.
+
+## Cell Pointer Events
+
+The `Datagrid` provides the following Events to handle mouse pointer interactions to the cells.
+
+* [CellEntered]() : Called when the mouse pointer enters the cell.
+* [CellHovered]() : Called when the mouse pointer is hovered over the cell.
+* [CellExited]()  : Called when the mouse pointer exits the cell.
+
+### CellEntered event
+This event will be triggered when the mouse pointer enters a cell in the DataGrid. It uses [DataGridCellEnteredEventArgs]() as its argument.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellEntered="dataGrid_CellEntered"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellEntered(object sender, DataGridCellEnteredEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellHovered event
+This event will be triggered when the mouse pointer hovers over a cell in the DataGrid. It uses [DataGridCellHoveredEventArgs]() as its argument.
+
+{% tabs %}
+{% highlight xaml tabtitle="MainPage.xaml" %}
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                   CellHovered="dataGrid_CellHovered"
+                   ItemsSource="{Binding OrderInfoCollection}" />
+{% endhighlight %}
+{% highlight c# tabtitle="MainPage.xaml.cs" %}
+private void dataGrid_CellHovered(object sender, DataGridCellHoveredEventArgs e)
+{
+    var rowIndex = e.RowColumnIndex.RowIndex;
+    var rowData = e.RowData;
+    var columnIndex = e.RowColumnIndex.ColumnIndex;
+    var column = e.Column;
+    var point = e.Point;
+}
+{% endhighlight %}
+{% endtabs %}
+
+### CellExcited event
```

Comment:
CellExited event

Reply:
![image](https://github.com/syncfusion-content/maui-docs/assets/113881396/bcf1cf91-7531-417e-98b3-99a13859ae46)
Had updated the title.